opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/point.py

```diff
@@ -101,6 +101,18 @@ class ExponentialHistogram:
         "opentelemetry.sdk.metrics.export.AggregationTemporality"
     )
 
+    def to_json(self, indent=4) -> str:
+        return dumps(
+            {
+                "data_points": [
+                    loads(data_point.to_json(indent=indent))
```

Comment:
Nice to see that this is consistent with the other implementations.

But I think it highlights the need for a `to_dict()` for these classes in general (separate PR) so we don't turn objects into dictionaries then into JSON and then into dictionaries.